src/compiler/types.ts

```diff
@@ -1212,11 +1212,11 @@ namespace ts {
          */
         emit(targetSourceFile?: SourceFile, writeFile?: WriteFileCallback): EmitResult;
 
-        getSyntacticDiagnostics(sourceFile?: SourceFile): Diagnostic[];
+        getOptionsDiagnostics(): Diagnostic[];

```

Comment:
Sure; but we don't have a semantic core to the parser, whereas we do have options that we give when parsing.